runtime/cere-dev/src/lib.rs

```diff
@@ -1401,7 +1403,12 @@ pub type SignedPayload = generic::SignedPayload<RuntimeCall, SignedExtra>;
 pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, RuntimeCall, SignedExtra>;
 
 /// Runtime migrations
-type Migrations = ();
+type Migrations = (
+	pallet_ddc_clusters::migrations::v3::MigrateToV3<Runtime>,
+	pallet_ddc_clusters::migrations::v2::MigrateToV2<Runtime>,
```

Comment:
`pallet_ddc_clusters::migrations::v2` should be removed from `runtime/cere-dev/src/lib.rs` as it has been executed in Devnet already

Reply:
<img width="1433" alt="image" src="https://github.com/user-attachments/assets/eef41565-62b9-4faf-9b74-a8dd605b488f">